docs/wiki/guides/current/Position-Hold-v4-6.md

```diff
@@ -0,0 +1,149 @@
+# Position Hold 4.6
+
+Betaflight 4.6 introduces Altitude Hold and Position Hold
+
+When used together, these features allow the pilot to hit a switch and the aircraft will hover at a fixed location and constant altitude, in Angle Mode. The sticks can then be used to ajust altitude and location. A GPS is essential, and a Magnetometer is strongly recommended.
+
+## Quick-Start
+
+- Include `ALTITUDE_HOLD` and `POSITION_HOLD` in your firmware build.
+- Ensure your GPS is working properly
+- Ideally, include a Magnetometer and verify that it is working properly
+- Set the hover throttle value accurately
+- Enable both Altitude Hold and Position Hold on the same Mode switch
+- Hover the quad and test carefully; the aircraft will be in Angle mode and stick responses will be different from usual.
```

Comment:
I've seen questions about how to tune angle-mode rates. is there a document and should it be linked here?

Reply:
Probably should have a document about Angle and Horizon modes, generally, where we could include tuning, rates and info about feedforward in Angle mode and how Horizon mode works.  Angle mode is a lot different now than it was.